src/ert/config/gen_kw_config.py

```diff
@@ -611,3 +625,18 @@ def calculate(self, x: float, arg: List[float]) -> float:
     "CONST": TransferFunction.trans_const,
     "RAW": TransferFunction.trans_raw,
 }
+
+
+TRANS_FUNC_ARGS: dict[str, List[str]] = {
```

Comment:
Perhaps we should consider not using the term ARGS (arguments).
To me, this is a dictionary which maps distribution types to a list of their respective `parameters`.
Is `DISTRIBUTION_PARAMETERS` perhaps a better name?